addons/packages/grafana/7.5.16/bundle/.imgpkg/images.yml

```diff
@@ -0,0 +1,18 @@
+---
+apiVersion: imgpkg.carvel.dev/v1alpha1
+images:
+- annotations:
+    kbld.carvel.dev/id: grafana/grafana:7.5.16
+    kbld.carvel.dev/origins: |
+      - resolved:
+          tag: 7.5.16
+          url: grafana/grafana:7.5.16
+  image: index.docker.io/grafana/grafana@sha256:9b7b1d9a1deadbe6fed74416d58db47f8af31b5cee214cfb659f89e6dda3f716
+- annotations:
+    kbld.carvel.dev/id: kiwigrid/k8s-sidecar:1.15.6
```

Comment:
@shyaamsn could you please help us understand why we should pin the version to 1.15.6?